### DISCUSSION TITLE
Help me plz with request contact

### DISCUSSION THREAD

**Discussion by chmerev** [aiogram/aiogram#748] [Q&A]
I have code like this to handle start message:

```
@dp.message_handler(commands=['start'])
async def send_welcome(message: types.Message):
  have_in_bd_contact = add_chat_name(message.from_user.full_name, message.chat.id)
  if have_in_bd_contact == "Written to the database":
    await message.answer(text="Hey! I registered you in my base", reply_markup=reply_keyboard.ReplyKeyboardMarkup(keyboard="1"))
```

I need the request_contact button to be highlighted. I have tried different methods already and it does not work out in any way. Help please) oh gods AIOGRAM

**Comment by Olegt0rr** [comment]
### We don't write code for other developers
If you want to learn how to write code yourself, then 

- learn the basics of Python
- explore the examples in the 'examples' folder
- read the Telegram API documentation
 
Further, if you are faced with a problem at the level of the programming language - contact the language learning communities. If you come across at the level of understanding the work of aiogram - attach the code and the problem that arises (with errors and a trackback or with a description of the wrong behavior).
